src/main/java/com/epam/tc/hw7/pages/JdiHomePage.java

```diff
@@ -0,0 +1,38 @@
+package com.epam.tc.hw7.pages;
+
+import com.epam.jdi.light.elements.common.Label;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.ui.html.elements.common.Icon;
+import com.epam.tc.hw7.Config;
+import com.epam.tc.hw7.forms.LoginForm;
+import jdi.entities.User;
+
+public class JdiHomePage extends WebPage {
+
+    public static final User ROMAN_IOVLEV = new User(
+        Config.getUserNameFromProperties(),
+        Config.getUserPasswordFromProperties(),
+        Config.getUserFullNameFromProperties());
```

Comment:
There can be many different users, they refer to the implementation of the tests, not the implementation of the page.

Reply:
Moved creation of user to AbstractTest